tests/suites/config.js

```diff
@@ -15,6 +15,10 @@ module.exports = [
 		}
 	},
 	image: `${__dirname}/balena-image.docker`,
+	// This is the name of a folder or file containing extra artifacts needed for the test suite
+	// E.g for the cm4-io-board provisioning the `secure-boot-msd` folder is required
+	// This artifact/artifact must be copied into the `suites` folder before running the test
+	artifacts: process.env.ARTIFACTS,
```

Comment:
@alexgg Can you add the same property to config files inside the test suites. With the move to GH actions, we will be moving to 3 different config files. One file for one test suite. You will find the files under each test suite folder.

Reply:
Just did @vipulgupta2048 why do we need so many `config.js` files that just repeat the same. They are not even used as it's the one in the leviathan/workspace that is used.

Couldn't we just remove them or add just a single `config.js.template`?

Having to keep them in sync serves no purpose imo.

Reply:
Leviathan's parallel test-running feature has been broken since we moved the core to the client. Previous cycle for the GH actions, we made a decision we did want the parallel test running back. To not spend half a cycle fixing and testing the feature in Leviathan, I made the easy decision to split the config.js into 3 files between each test suite which launches three 3 separate test jobs. This resulted in test runs which are clean, atomic and independent.

In Jenkins, 
We delete the config.js in the leviathan/workspace and replace it with the config.js from meta-balena and run the test from it sequentially.

In GH actions, 
We run the test directly from the config.js present in test suite. 

So the test does make use of them and we are getting the benefit of faster, more retryable tests by having 3 copies of almost same config.js 

Downside, we need to keep them in sync.

Reply:
@vipulgupta2048 Let's follow up in https://balena.zulipchat.com/#narrow/stream/345889-balena-io.2Fos/topic/Levithan.20config.2Ejs as this has nothing to do with this PR.